lib/jekyll-admin/directory.rb

```diff
@@ -0,0 +1,62 @@
+module JekyllAdmin
+  class Directory
+    attr_reader :path
+
+    include Enumerable
+    include JekyllAdmin::URLable
+    include JekyllAdmin::APIable
+
+    TYPE = 'directory'
+
+    def initialize(path, args={})
```

Comment:
I *believe* you're trying to use named arguments here... this can be simplified to:

```ruby
def initialize(path, base: nil, content_type: nil, splat: nil)
  @base = Pathname.new base
  @content_type = content_type
  @splat = Pathname.new splat
  @path = Pathname.new path
end
```